Firmware/MotorControl/motor.hpp

```diff
@@ -187,6 +193,7 @@ class Motor {
     DRV8301_FaultType_e drv_fault_ = DRV8301_FaultType_NoFault;
     DRV_SPI_8301_Vars_t gate_driver_regs_; //Local view of DRV registers (initialized by DRV8301_setup)
     float thermal_current_lim_ = 10.0f;  //[A]
+    float inverter_temp_ = 20.0f;
```

Comment:
I would say 0.0f would be a better default. Looks more "uninitialised" than 20.

Reply:
or NaN? Could surprise one or the other user but seems to me like the cleanest default.